x-pack/plugins/security_solution/server/lib/detection_engine/signals/utils.ts

```diff
@@ -381,7 +381,7 @@ export const errorAggregator = (
   ignoreStatusCodes: number[]
 ): BulkResponseErrorAggregation => {
   return response.items.reduce<BulkResponseErrorAggregation>((accum, item) => {
-    if (item.create?.error != null && !ignoreStatusCodes.includes(item.create.status)) {
+    if (item.create?.error?.reason != null && !ignoreStatusCodes.includes(item.create.status)) {
```

Comment:
If I understand correctly, reason can be `undefined` now, which leads to situations where it can't be used as object key, as in this case.
With checking only reason existence, we can potentially miss some errors.
I would propose something like this:
```typescript
    if (item.create?.error != null && !ignoreStatusCodes.includes(item.create.status)) {
      const reason = item.create.error.reason ?? 'unknown';
      if (accum[reason] == null) {
        accum[reason] = {
          count: 1,
          statusCode: item.create.status,
        };
      } else {
        accum[reason] = {
          count: accum[reason].count + 1,
          statusCode: item.create.status,
        };
      }
    }
```
If reason is `undefined`, it still will be captured in this aggregator

Reply:
@vitaliidm good call! I applied your suggestion in [elastic/kibana@`7ee8cd1` (#145416)](https://github.com/elastic/kibana/pull/145416/commits/7ee8cd152c37d13dd3a5f2ac41abb08518b0ad1b)